lock-api/versions.lock

```diff
@@ -44,11 +46,15 @@
                 "com.palantir.atlasdb:atlasdb-commons"
             ]
         },
+        "com.palantir.atlasdb:timestamp-api": {
+            "project": true
+        },
```

Comment:
We could make a new timelock project. I was thinking about doing some moving around / reorganizing but would prefer to do that in a follow up PR once everything is in place. Will add it to the burndown